cvmfs/object_fetcher.h

```diff
@@ -0,0 +1,387 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef OBJECT_FETCHER_H
+#define OBJECT_FETCHER_H
+
+#include <string>
+
+#include "manifest.h"
+#include "manifest_fetch.h"
+#include "history_sqlite.h"
+#include "catalog.h"
+
```

Comment:
[Fixed](https://github.com/reneme/cvmfs/commit/3b4b6ac4950f297d6ec280d6e7eb7ca58a4e0c36) for newly added files.... Doing this consistently everywhere is [left for later](https://sft.its.cern.ch/jira/browse/CVM-740)...